models/models.reviews.js

```diff
@@ -34,3 +34,26 @@ exports.selectReviewById = (reviewId) => {
 		return reviews[0];
 	});
 };
+
+exports.updateReviewById = (newVotes, reviewId) => {
+	// Check for 204
+	if (newVotes.inc_votes === 0) {
+		return Promise.resolve();
+	}
+
+	// Check for 404
+	return this.selectReviewById(reviewId)

```

Comment:
I dont know if you need to create this chain where you check to see if the review exists here, you can handle this effectively by looking at the return object, which would mean you didn't have to create this sequential code